cfgov/cfgov/tests/test_urls.py

```diff
@@ -133,12 +133,20 @@ def test_handle_error_attribute_error(self, mock_render):
 
     @mock.patch('cfgov.urls.render')
     def test_handle_error(self, mock_render):
-        request = self.factory.get('/test')
+        request = self.factory.get('/Test')
         urls.handle_error(404, request)
         mock_render.assert_called_with(
             request, '404.html', context={'request': request}, status=404
         )
 
+    @mock.patch('cfgov.urls.render')
+    def test_handle_404_error(self, mock_render):
+        request = self.factory.get('/test')
+        urls.handle_404_error(404, request)
+        mock_render.assert_called_with(
+            request, '404.html', context={'request': request}, status=404
+        )
+
```

Comment:
Could you add another test of the case-insensitivity handling in `urls.handle_404_error`? I think something like:

```python
    def test_handle_404_error(self):
         request = self.factory.get('/TEst')
         response = urls.handle_404_error(404, request)
         self.assertRedirects(response, '/test')
```

Reply:
Done